.github/workflows/create-release-branch.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       # Increment the version as desired locally, without actually committing anything.
       - name: Locally increment version
         run: |
-          npm --no-git-tag-version version prerelease --preid alpha
+          npm --no-git-tag-version version prerelease --preid beta
```

Comment:
NOTE: In its current state, branches and releases will be versioned as `5.0.0-beta.N` instead of `5.0.0-alpha.N`. We _could_ make it so that they release as `5.Y.Z`, but my intuition is that we don't want to burn a bunch of early minor version numbers on beta releases any more than we wanted to with alphas.